### PR TITLE
Add acceptance test for video playback

### PR DESCRIPTION
**Pull request by z4y4ts**

### CODE REVIEW

**Comment by z4y4ts** [comment]
@polesye @TamoshaytisV would really appreciate your review

File: video_xblock/static/html/student_view.html [hunk 1]
```diff
@@ -1,6 +1,7 @@
 {% load i18n %}
 <h3 class="hd hd-2">{{display_name}}</h3>
 <iframe
+    id="xblock-video-player"
```
Comment (polesye) [inline video_xblock/static/html/student_view.html]: Id has to be unique per page, are you sure it correct change?

File: test_requirements.txt [hunk 2]
```diff
@@ -1,10 +1,16 @@
+bok-choy==0.4.3
```
Comment (polesye) [inline test_requirements.txt]: where do you use it? If you're not, so why do you need it?

File: video_xblock/static/html/student_view.html (hunk 1 above)
Comment (z4y4ts) [inline video_xblock/static/html/student_view.html]: Good point, thank you!

**Pull request merged**